nilearn/plotting/img_plotting.py

```diff
@@ -143,11 +143,13 @@ def _plot_img_with_bg(img, bg_img=None, cut_coords=None,
                    'Tip: Use np.nanmax() instead of np.max().')
         warnings.warn(nan_msg)
 
-    if isinstance(cut_coords, numbers.Number) and display_mode == 'ortho':
+    if (isinstance(cut_coords, numbers.Number) and
```

Comment:
Please add test related to this in nilearn/plotting/tests/test_img_plotting.py. Sorry for late on this.

There are already tests related to display_mode='ortho' where you could inspire from ```test_invalid_in_display_mode_cut_coords_all_plots```.